p/themes/base-theme/frss.css

```diff
@@ -2743,6 +2735,7 @@ html.slider-active {
 	}
 
 	#nav_entries {
+		display: table !important;
```

Comment:
When possible, I am happier when `!important` can be avoided, for instance with a more precise selector